src/main/java/exercise31/KarvonenHeartRate.java

```diff
@@ -13,19 +14,24 @@ public int formula(int age, int restingHR, double intensity) {
         return bpm;
     }
 
+    //Naming / data-type?
     public String[] checkDifferentIntensities(int age, int restingHR) {
+        //Shorter way to do addition?
         for (double intensity = 55; intensity <= 95; intensity = intensity + 5) {
+            //Brackets and whitespace?
             table[index] = (formula(age, restingHR, intensity) + "");
 
+            //Why the additional counter?
             if(index < table.length){
                 index++;
             }
 
         }
-
+        //What about printing out the whole table?
         return table;
     }
 
+    //What's this main method doing?
     public static void main(String[] args) {
```

Comment:
Probably just left over from testing but this main method doesn't actually seem to do anything...?